packages/react/src/SignInForm.tsx

```diff
@@ -78,6 +79,8 @@ export function SignInForm(props: SignInFormProps): JSX.Element {
           );
         } else if (memberships) {
           return <ProfileForm login={login} memberships={memberships} handleAuthResponse={handleAuthResponse} />;
+        } else if (props.projectId === 'new') {
```

Comment:
Am I going to be able to cause mischief if I name my project "new" or edit the redirect URL to manually set the projectID to 'new'?

Reply:
Good question

* If you have an existing user account
  * You still have to sign-in with your current credentials
  * You could have arrived at the URL by a legitimate path
  * So I think that's fine
* If you do not have an existing account
  * You still have to sign-in, so you should never get to this point in the code
  * So I think that's fine

Perhaps a bigger question is whether this belongs inside the `<SignInForm>` component at all, and whether it makes more sense 100% in our `app` project.  The answer to that is "probably", but it would increase the complexity of the `<SignInForm>`, so I'm inclined to leave it where it is.